src/ClusterManager.php

```diff
@@ -20,4 +20,26 @@ public function getBootstrapScript(string $cluster)
             url: '/v1/cluster/'. $cluster .'/bootstrap-url',
         );
     }
+
+    public function putSecret(string $cluster, string $namespace, string $name, string $key, $payload)
+    {
+        try {
+            return $this->fetch(
+                type: 'put',
+                url: sprintf("/cluster/%s/secret/%s/%s/%s", $cluster, $namespace, $name, $key),
+                payload: $payload,
```

Comment:
I think this will default to Content-Type `application/json`? This will make express attempt JSON decode, which will usually be wrong, and will probably seal something incorrect like a stringified object reference.
The Content-Type wants to be `application/octet-stream` for this endpoint; this is uninterpreted data. I'm not sure if/how the PHP service client supports setting content type?